packages/types/src/company.ts

```diff
@@ -3,14 +3,15 @@ import { z } from "zod"
 export const CompanySchema = z.object({
   id: z.string().uuid(),
   createdAt: z.date(),
-  name: z.string().max(50),
-  description: z.string(),
-  phone: z.string().nullable(),
-  email: z.string(),
-  website: z.string(),
+  name: z.string().max(50).min(1),
```

Comment:
Shorthand with better error msg in zod is .nonempty()

Reply:
niiiiice thanks!

Reply:
`@deprecated — Use z.string().min(1) instead.` :(